addons/website_payment/models/res_currency.py

```diff
@@ -0,0 +1,13 @@
+from odoo import api, fields, models
+from odoo.addons.payment_razorpay.const import SUPPORTED_CURRENCIES as RAZORPAY_SUPPORTED_CURRENCIES
+
+
+class ResCurrency(models.Model):
+    _inherit = 'res.currency'
+
+    is_razorpay_supported_currency = fields.Boolean(compute='_compute_is_razorpay_supported_currency')
```

Comment:
It would be better to create field in res_company `is_razorpay_supported_company`